confluent-connector-github-demo/build.gradle

```
@@ -0,0 +1,57 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath "gradle.plugin.com.github.jengelman.gradle.plugins:shadow:7.0.0"
+    }
+}
+
+plugins {
+    id "java"
+    id "idea"
+    id "eclipse"
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_17
+    targetCompatibility = JavaVersion.VERSION_17
+}
+version = "0.0.1"
+
```

```suggestion
application {
   mainClass = "clients.GitHubPrRatio"
 }
```
Hopefully it has enough space it shouldn't bleed into the `repositories` block